fs/dafsserver/dafsserver.cpp

```diff
@@ -5575,7 +5580,8 @@ class CRemoteFileServer : implements IRemoteFileServer, public CInterface
                     }
                     if (exception)
                     {
-                        EXCLOG(exception, "CRemoteFileServer (secure)");
+                        if (exception->errorCode() != DAFSERR_serveraccept_fail_portcheck)
+                            EXCLOG(exception, "CRemoteFileServer (secure)");
```

Comment:
I'm a bit concerned this will mask the problem we saw on cloud with the excessive logging.
i.e. it seemed to be in a tight loop logging continuously without corresponding frequent connections.

If we suppress the logging altogether we wouldn't know ..

Perhaps keep a count of # of exceptions and log every minute ?

Reply:
added some code - do you mean like this in commit 3 ?
Please re-review

Reply:
yes, but to be consistent, worth also doing in other cases, e.g. at line 5536 and 5639.
Can use same counter and call a common helper though to avoid duplicating code.

Worth logging one of exceptions each time the time limit is hit too, which may help diagnose the underlying type of issue.

Reply:
Added code to line 5639
Line 5536 is not a secure accept, so nothing to fail/throw.
Exception has no new/addl info over a warning message.